### PR TITLE
Update techCompanies.js

### PR DESCRIPTION
**Pull request by Victorayo**
Added Trove Finance to the tech companies Json file.

Kindly merge @Chaos10001 @OSCA-Ado-Ekiti

### CODE REVIEW

**Comment by Victorayo** [comment]
> Added Trove Finance to the tech companies Json file.
> 
> Kindly merge @Chaos10001 @OSCA-Ado-Ekiti

In reference to issue  #119

File: src/techCompanies.js
Comment (gray-adeyi) [inline src/techCompanies.js:1016]: Please also include it also in the `README.md` file

**Review by gray-adeyi — CHANGES_REQUESTED** [review]

File: src/techCompanies.js
Comment (Victorayo) [reply src/techCompanies.js:1016]: Have added it @gray-adeyi . You will find the PR as #124

Comment (gray-adeyi) [reply src/techCompanies.js:1016]: Great! I don't have merge access to this repository. You pr will be merged soonest

Comment (Victorayo) [reply src/techCompanies.js:1016]: Alright boss, thanks. @gray-adeyi 

Waiting for @Chaos10001 and @OSCA-Ado-Ekiti to merge

**Pull request merged**
